include/teb_local_planner/g2o_types/edge_velocity.h

```diff
@@ -268,6 +268,123 @@ class EdgeVelocityHolonomic : public BaseTebMultiEdge<3, double>
 };
 
 
+/**
+ * @class EdgeVelocityAbsolute
+ * @brief Edge defining the cost function for limiting the absolute translational velocity.
+ *
+ * The edge depends on three vertices \f$ \mathbf{s}_i, \mathbf{s}_{i-1}, \Delta T_i \f$ and minimizes: \n
+ * \f$ \min \textrm{penaltyInterval}( [max_v]^T ) \cdot weight \f$. \n
+ * \e max_v denotes the maximum velocity of the robot. \n
+ * \e weight can be set using setInformation(). \n
+ * \e penaltyInterval denotes the penalty function, see penaltyBoundToInterval(). \n
+ * The dimension of the error / cost vector is 1: the first component represents the velocity.
+ * @see TebOptimalPlanner::AddEdgesVelocity
+ * @remarks Do not forget to call setTebConfig()
+ */
+class EdgeVelocityAbsolute : public BaseTebMultiEdge<1, double>
+{
+public:
+  /**
+   * @brief Construct edge.
+   */
+  EdgeVelocityAbsolute()
+  {
+    this->resize(3); // Since we derive from a g2o::BaseMultiEdge, set the desired number of vertices
+  }
+
+  /**
+   * @brief Actual cost function
+   */
+  void computeError()
+  {
+    ROS_ASSERT_MSG(cfg_, "You must call setTebConfig on EdgeVelocityAbsolute()");
```

Comment:
cool idea, but assert will be disabled if we compile with NDEBUG (which is the default for our release)
https://docs.ros.org/diamondback/api/rosconsole/html/assert_8h_source.html

maybe ROS_FATAL_COND() would do it?

Reply:
Followed the "style guide" of the other existing edges here.